js/app.js

```diff
@@ -8,6 +8,163 @@ const menuBody = document.getElementById('mobile-menu-body');
+    images: ['tonic-2-project.jpg', 'Snapshoot-Portfolio-3.png'],
+  },
+};
+
+function modalPopup(data) {
+  menuBody.style.overflow = 'hidden';
+  tonicProjectModal.style.display = 'block';
+  tonicProjectModal.innerHTML += `
+    <div class='tonic-project-modal-content' id='tonic-project-modal'>
+    <div class='tonic-project-content'>
+        <div class='tonic-project-modal-header'>
+            <h2 class='tonic-h2'>${data[0]}</h2>
+            <span class='tonic-project-close' onClick="document.getElementById('modal-div').style.display = 'none';
+            document.getElementById('mobile-menu-body').style.overflow ='visible';">&times;</span>
+        </div>      
+        <ul class='works-headers tonic-project-modal-header-techs'>
+            <li class='title-header' style='margin-right: 50px;'>${data[1]}</li>
+            <li><img src='images/works-dot.PNG' style='margin-right: 10px;' class='dot-separator' alt='Dot separator'></li>
+            <li class='title-header-general'  style='margin-right: 23px;'>${data[2]}</li>
+            <li><img src='images/works-dot.PNG' style='margin-right: 13px;' class='dot-separator' alt='Dot separator'></li>
+            <li class='title-header-general'>2015</li>
+        </ul>
+        <div class='tonic-project-modal-body'>
+            <figure class='img-wide-screen-modal'>
+                <img src='images/${data[5][0]}' class='works-image style-image small-screen-view' alt='Tonic App Screenshot'>
+                <img src='images/${data[5][1]}' class='wide-screen-view modal-pic-full' alt='Tonic project image Screenshot'>
+            </figure>
+        </div>
+        <section class='project-aside'>
+            <div class='project-aside-description'>
+                <p class='project-description-tonic small-screen-view-des'>
+                 ${data[3][0]}
+                </p>
+                <p class='project-description-tonic full-screen-view-des'>
+                ${data[3][1]}
+                </p>
+            </div>
+            <div class='project-aside-languages'>
+                <section class='project-languages-tonic'>
+                    <ul class='my-work-languages-tonic' id='language-list'>
+                      <li class='coding-languages'>${data[4][0]}</li>
+                      <li class='coding-languages'>${data[4][1]}</li>
+                      <li class='coding-languages'>${data[4][2]}</li>
+                    </ul>
+                </section>
+                <div class='tonic-project-modal-footer'>
+                    <button class='tonic-see-live'><span class='tonic-see-live-text'>See live</span> <img class='tonic-see-live-img' src='./images/live.JPG' alt='See source on Github' srcset=''></button>
+                    <button class='tonic-see-live'><span class='tonic-see-live-text'>See Source</span> <img class='tonic-see-live-img' src='./images/cat.png' alt='See source on Github' srcset=''></button>
+                </div>
```

Comment:
- [ ] Please implement the button interactions for the model buttons.
![image](https://user-images.githubusercontent.com/83803414/195918252-91af4469-572f-496c-86a4-e7809135adb9.png)